src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IConstructorBodyOperation.cs

```diff
@@ -281,5 +432,606 @@ class C
             var node1 = tree.GetRoot().DescendantNodes().OfType<ConstructorDeclarationSyntax>().Single();
             Assert.Null(model.GetOperation(node1));
         }
+
+        [CompilerTrait(CompilerFeature.Dataflow)]
+        [Fact]
+        public void ConstructorBody_09()
+        {
+            string source = @"
+class C
+{
+    public C() : base()
```

Comment:
Is constructor chaining `: this()` going to be done in a separate PR?

Reply:
That shouldn't affect the flow graph, its just a different invocation in the initializer. I will still cover it in the new test I plan to add for your above suggestion.

Reply:
Agree it shouldn't be meaningfully different. Still would like to have at least one test to ensure we exercise the code path.